### ISSUE TITLE
Code Review tools

### ISSUE THREAD

**Issue by m1gu3l** [Microsoft/vso-intellij#99]
Since there is support for TFVC, it would be great to start thinking on support of code review tools, as mentioned in #17

**Comment by leantk** [comment]
Thanks for the feedback @m1gu3l! At the moment we don't have this planned as a feature but I'll add this as an enhancement and it then will be considered for future work. 

Thanks again,
-Leah

**Comment by kelong** [comment]
Any update on this? Thanks

**Comment by sme81** [comment]
any renewed plans for scheduling this enhancement?

**Comment by Rudokus** [comment]
Yes me 2, I am really looking forward to jump completely on the intellij stack and get rid of visual studio.

**Comment by Cselt** [comment]
Any plans on this? I would really love to use intellij / webstorm to do the CR. 
I think Intellij has a really powerful compare view which I would like to use for code reviews.

**Comment by wpmichel** [comment]
Bumping

**Comment by NickRimmer** [comment]
It would be a great feature if we will have this code review tool for Azure DevOps git repos.

**Comment by herbatnik** [comment]
🙏

**Comment by 1fexd** [comment]
Is this feature planned in the (near) future?

**Comment by ForNeVeR** [comment]
It is available [there](https://github.com/JetBrains/intellij-community/tree/6051a2be895376d957bab48a1868cb5b8c7650fe/plugins/github/src/org/jetbrains/plugins/github/pullrequest), I think.

**Comment by Jonatha1983** [comment]
@nathanpovo That is what I did - it took me 2-3 months and still working on it

I uploaded the plugin it is in early version but should work for Azure Devops service (and soon for server): https://plugins.jetbrains.com/plugin/22319-azd
